engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -986,11 +984,8 @@ public FilteredTable(final TrackingRowSet currentMapping, final QueryTable sourc
                 final WhereFilter[] filters) {
             super(source.getDefinition(), currentMapping, source.columns, null, null);
             this.source = source;
-            this.filters = filters;
             for (final WhereFilter f : filters) {
-                if (f instanceof LivenessReferent && f.isRefreshing()) {
-                    manage((LivenessReferent) f);
-                }
+                addParentReference(f);
```

Comment:
Really, the thing that *should* own these is the `WhereListener`. But maybe we don't want to mess with it?